doc/cephfs/cephfs-shell.rst

```diff
@@ -508,6 +506,34 @@ Options :
 
   --max_files MAX_FILES    Set total number of files under this directory tree
 
+CephFS Shell Configuration File
+===============================
+By default, CephFS Shell looks for ``cephfs-shell.conf`` at the path provided
+in the environment variable ``CEPHFS_SHELL_CONF`` and then in user's home
+directory (``~/.cephfs-shell.conf``).
+
+Right now, CephFS Shell inherits all its options from its dependency ``cmd2``.
+Therefore, these options might vary with the version of ``cmd2`` installed on
+your system. Refer to ``cmd2`` docs for a description of these options.
```

Comment:
Add link to cmd2 doc

Reply:
I left that out on purpose since conf options can vary significantly with cmd2.